pkg/kubebench/scanner.go

```diff
@@ -102,18 +102,22 @@ func (s *Scanner) Scan(ctx context.Context, nodeName string, target string, wg *
 		return
 	}
 
-	node, err = s.clientset.CoreV1().Nodes().Get(ctx, kubeBenchPod.Spec.NodeName, meta.GetOptions{})
-
 	return
 }
 
-func (s *Scanner) prepareKubeBenchJob(nodeName string, target string) *batch.Job {
+func (s *Scanner) prepareKubeBenchJob(node core.Node) *batch.Job {
+	target := "node"
+	if _, ok := node.Labels[masterNodeLabel]; ok {
+		target = "master"
+	}
 	return &batch.Job{
 		ObjectMeta: meta.ObjectMeta{
 			Name:      uuid.New().String(),
 			Namespace: kube.NamespaceStarboard,
-			Labels: map[string]string{
-				"app": "kube-bench",
+			Labels: labels.Set{
+				"app.kubernetes.io/name": "starboard-cli",
+				kube.LabelResourceKind:   string(kube.KindNode),
+				kube.LabelResourceName:   node.Name,
```

Comment:
I wonder if in addition we should add a label to indicate that it's a kube-bench job? something like `"starboard-app": "kube-bench"` maybe?  (also for pod-spec and for other kinds of tool)

Reply:
Yes. That's a very good point. I'll take it as a separate issue to review the labels to identify scan job and the underlying tools.
I think it's less relevant for Starboard CLI, but the Operator has no clue which scanner we're running in the starboard namespace.